server/etcdserver/apply/uber_applier.go

```diff
@@ -103,21 +102,22 @@ func (a *uberApplier) RestoreAlarms() {
 	}
 }
 
-func (a *uberApplier) Apply(r *pb.InternalRaftRequest, shouldApplyV3 membership.ShouldApplyV3) *applyResult {
+func (a *UberApplier) Apply(r *pb.InternalRaftRequest, shouldApplyV3 membership.ShouldApplyV3) *ApplyResult {
 	return a.applyV3.WrapApply(context.TODO(), r, shouldApplyV3, a.dispatch)
 }
 
 // This function
-func (a *uberApplier) dispatch(ctx context.Context, r *pb.InternalRaftRequest, shouldApplyV3 membership.ShouldApplyV3) *applyResult {
+func (a *UberApplier) dispatch(ctx context.Context, r *pb.InternalRaftRequest, shouldApplyV3 membership.ShouldApplyV3) *ApplyResult {
 	op := "unknown"
-	ar := &applyResult{}
+	ar := &ApplyResult{}
 	defer func(start time.Time) {
-		success := ar.err == nil || ar.err == mvcc.ErrCompacted
-		applySec.WithLabelValues(v3Version, op, strconv.FormatBool(success)).Observe(time.Since(start).Seconds())
-		warnOfExpensiveRequest(a.lg, a.warningApplyDuration, start, &pb.InternalRaftStringer{Request: r}, ar.resp, ar.err)
-		if !success {
-			warnOfFailedRequest(a.lg, start, &pb.InternalRaftStringer{Request: r}, ar.resp, ar.err)
-		}
+		op += " "
+		//		success := ar.Err == nil || ar.Err == mvcc.ErrCompacted
```

Comment:
TODO?

Reply:
Sorry - seems I didn't exported the newest version. PTAL